components/runtime/src/ts/discord/dapi.ts

```diff
@@ -103,6 +103,8 @@ export interface CreateMessageFields {
     allowedMentions?: AllowedMentions;
 
     components?: IComponent[],
+
+    reply_to_message_id?: string;
```

Comment:
should be in camelCase

Reply:
Pushed a fixup, that one somehow eluded my eagle-like eyes.